### PR TITLE
Links: lisp-lang.org

### PR DESCRIPTION
**Pull request by vindarel**
I added lisp-lang once, we could link to it more !

### CODE REVIEW

File: ides/emacs-setup.html [hunk 1]
```diff
@@ -5,15 +5,51 @@
...
+<h3>With Portacle</h3>
+
+<a href="https://shinmera.github.io/portacle/">Portacle</a> is a
+portable and multiplatform Common Lisp development environment. It
+packages SBCL, Emacs25, Slime, Quicklisp and Git altogether. There is no
+installation required, so it's the easiest way to go.
+
+<h3>Install GNU/Emacs</h3>
+
+<p>In order to use emacs, there are two variants; XEmacs and GNU
```
Comment (pnathan) [inline ides/emacs-setup.html]: hmm. care to wordsmith it? xemacs is essentially dead these days as far as I know.

File: ides/emacs-setup.html [hunk 2]
```diff
@@ -5,15 +5,51 @@
...
+the <a href="http://www.gnu.org/software/emacs/">GNU</a>. Windows,
+Linux, and OSX versions are available. It&#8217;s also available via
+most Linux distributions package repositories. The most recent version
+is 25; 23 still works well with the Superior Lisp
+Interaction Mode for Emacs, a.k.a SLIME.</p>
+
+<p>New users can find many resources online for Emacs, including a
+well-written help system! An emacs tutorial can be
+found <a href="http://www.gnu.org/software/emacs/manual/html_mono/emacs.html">here</a>,
+and the GNU FAQ is
```
Comment (pnathan) [inline ides/emacs-setup.html]: s/GNU FAQ/GNU emacs FAQ/

Comment (vindarel) [inline ides/emacs-setup.html]: I edited as you requested in this PR: https://github.com/pnathan/articulate-common-lisp/pull/60

sorry I'm unclear about my PRs, this one is supposed to be only about lisp-lang.org, but it counted the commits I did in master (and shouldn't have). You already merged the commits about Emacs: https://github.com/pnathan/articulate-common-lisp/pull/57

**Comment by vindarel** [comment]
Looks like this is merge-able !

**Pull request merged**